### PR TITLE
Init even with no bundle being available

### PR DESCRIPTION
**Pull request by scorphus**
i.e. before rbenv sets up bundle

### CODE REVIEW

File: bundler.fish [hunk 1]
```diff
@@ -1,7 +1,6 @@
 function init --on-event init_bundler
   if test -z (which bundle ^ /dev/null)
```
Comment (bpinto) [inline bundler.fish]: `if not available bundle`

File: bundler.fish [hunk 2]
```diff
@@ -57,7 +56,11 @@ function init --on-event init_bundler
   end
 
   function __is_a_bundled_executable
-    set -l bindir (command bundle exec ruby -e "puts Gem.bindir")
-    test -f "$bindir/$argv"
+    if which bundle > /dev/null 2>&1
```
Comment (bpinto) [inline bundler.fish]: `if not available bundle`

**Comment by bpinto** [comment]
I like that it's not order dependent, BUT, it will always echo "bundler: Install bundle first!".

I think it's okay to remove this initial check though.

**Comment by scorphus** [comment]
Yeah, I agree with you.

File: bundler.fish (hunk 2 above)
Comment (scorphus) [inline bundler.fish]: Thanks!

**Comment by scorphus** [comment]
I like that `available` _trick_ :+1:

**Comment by bpinto** [comment]
Thanks @scorphus

**Pull request merged**